valhalla/midgard/logging.h

```diff
@@ -44,7 +46,8 @@ class Logger {
 };
 
 // statically get a logger using the factory
-Logger& GetLogger(const LoggingConfig& config = {{"type", "std_out"}, {"color", "true"}});
+Logger& GetLogger(const LoggingConfig& config = {{"type", "std_out"},
```

Comment:
the only thing im still thinking about here is, should we also, in the stdout factory function disable color if its not a tty? so over in the source file we have the stdout loggers constructor which blindly listens to whatever you put in the config. the change we've made here only helps for programs that dont initialize the logger from the config iirc

Reply:
That's a good question.  By design, I made this so that if *no* logging config is provided, then we autodetect, but if the user supplies something, we use that, assuming they know what they want.

I now see that `valhalla_build_config` generates a logging config, and it contains `color: true`.

Looking at the defaults in `valhalla_build_config`, they don't seem all that useful (e.g. the logging filename is `path_to_some_file.log`).  Would it make sense to simply remove them from the default generated config?

Reply:
The smallest change here would be to just detect the `color` setting if it's not supplied.  Then we could remove just that from `valhalla_build_config` and minimize the change overall.